File: tags/NHI1-0.16/theLink/example/java/MyClient.java
/**
 *  \file       theLink/example/java/MyClient.java
 *  \brief      \$Id$
 *  
 *  (C) 2009 - NHI - #1 - Project - Group
 *  
 *  \version    \$Rev$
 *  \author     EMail: aotto1968 at users.berlios.de
 *  \attention  this software has GPL permissions to copy
 *              please contact AUTHORS for additional information
 */
package example;
import javamsgque.*;
class MyClient extends MqS {
  public static void main(String[] argv) {
    MyClient c = new MyClient();
    try {
      c.LinkCreate(argv);
      c.SendSTART();
      c.SendEND_AND_WAIT("HLWO", 5);
      System.out.println(c.ReadC());
    } catch (Throwable e) {
      c.ErrorSet(e);
    }
    c.Exit();
  }
}

File: tags/NHI1-0.17/theLink/example/php/MyClient.php
<?php
#+
#:  \file       theLink/example/php/MyClient.php
#:  \brief      \$Id$
#:  
#:  (C) 2010 - NHI - #1 - Project - Group
#:  
#:  \version    \$Rev$
#:  \author     EMail: aotto1968 at users.berlios.de
#:  \attention  this software has GPL permissions to copy
#:              please contact AUTHORS for additional information
#:

$srv = new MqS();
try {
  $srv->ConfigSetName("MyClient");
  $srv->LinkCreate($argv);
  $srv->SendSTART();
  $srv->SendEND_AND_WAIT("HLWO");
  print($srv->ReadC());
} catch (Exception $ex) {
  $srv->ErrorSet($ex);
}
$srv->Exit();

?>
